tools/segmetrics/segmetrics.xml

```diff
@@ -13,24 +13,28 @@
    $unzip
    $is_seg_unique
    $is_gt_unique
-   $measures.dice
-   $measures.seg
-   $measures.jc
-   $measures.ji
-   $measures.ri
-   $measures.ari
-   $measures.hsd_sym
-   $measures.hsd_e2a
-   $measures.hsd_a2e
-   $measures.nsd
-   $measures.o_hsd_sym
-   $measures.o_hsd_e2a
-   $measures.o_hsd_a2e
-   $measures.o_nsd
-   $measures.fs
-   $measures.fm
-   $measures.fp
-   $measures.fn
+   #for $m in $measures
+       #set $kwargs = ""
+       #set $suffix = ""
+       #set $is_distance = False
+       #if str($m.measure_type.measure_type_selector) == "ISBIScore":
+           #set $kwargs = "min_ref_size=" + str($m.measure_type.min_ref_size) + ", "
+       #elif str($m.measure_type.measure_type_selector) == "Hausdorff":
+           #set $kwargs = "mode='" + str($m.measure_type.mode) + "', quantile=" + str($m.measure_type.quantile) + ", "
+           #set $is_distance = True
+       #elif str($m.measure_type.measure_type_selector) == "NSD":
+           #set $is_distance = True
+       #end if
+       #if $is_distance:
+           #if $m.measure_type.object_based:
+               #set $suffix = ".object_based()"
+           #end if
+       #end if
+       #if str($m.measure_type.aggregation) != "":
+           #set $kwargs = "aggregation='" + str($m.measure_type.aggregation) + "', " + $kwargs
+       #end if
+       "sm.${m.measure_type.measure_type_selector}(${kwargs})${suffix}"

```

Comment:
can we use single quotes whenever possible, this is a bit more secure

Reply:
I have changed this in 1e5865766fe9b9ec951cf76da33c13827c9ddaa7.

Anything else to be done before merging?